packages/nouns-contracts/contracts/governance/NounsDAOV3Votes.sol

```diff
@@ -146,7 +145,13 @@ library NounsDAOV3Votes {
         uint8 support,
         string calldata reason
     ) external {
-        emit VoteCast(msg.sender, proposalId, support, castVoteInternal(ds, msg.sender, proposalId, support), reason);
+        emit VoteCast(
```

Comment:
I think it's useful to have an emit that includes the client ID. let's discuss whether we add it here or add a new event.
I know we've made sure to be compatible with GovernorBravo with these events so far, I'm just not convinced that's important anymore.

Reply:
agree we should add the event
and we need to think about whether to keep compatibility for gov bravo